### ISSUE TITLE
Support Scheduled Scaling in AWS Autoscaling Groups

### ISSUE THREAD

**Issue by kurron** [hashicorp/terraform#4211]
It appears that as of Terraform v0.6.6 we cannot schedule scaling of our autoscaling groups.  The use case is that we have some test instances that don't need to be running at night so we want to shut them off.  Currently, you have to use the AWS CLI to accomplish this.  Here is a snippet of a Bash file to illustrate what I mean:

```bash
# set the capacity to 0 during normal off hours: 5PM to 8AM UTC every day
SCALE_DOWN='aws autoscaling put-scheduled-update-group-action --auto-scaling-group-name ecs-autoscaling-experiment --scheduled-action-name scaledown-schedule-daily --recurrence "0 17 1-31 1-12 1-7" --min-size 0 --max-size 0 --desired-capacity 0'
echo $SCALE_DOWN
eval $SCALE_DOWN
```
It would be great if Terraform could support this capability.

**Comment by stack72** [comment]
I have a WIP on this - will open a PR later tonight / tomorrow morning

**Comment by stack72** [comment]
I believe this is a Duplicate of #2367

**Comment by jen20** [comment]
This has landed in #4256! Thanks @stack72!
